back/engines/free/polls/spec/policies/response_policy_spec.rb

```diff
@@ -88,4 +88,10 @@
       expect(scope.resolve.size).to eq 0
     end
   end
+
+  context 'for blocked user' do
+    let(:user) { create(:user, block_end_at: 5.days.from_now) }
+
+    it_behaves_like 'policy for blocked user', show: false
+  end
```

Comment:
Test to confirm that creating a poll response (submitting a poll) is blocked when user is blocked.